src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+import duke.Task;
+import duke.Deadline;
+import duke.Todo;
+import duke.Event;
+import duke.DukeException;
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    private static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+
+    public static void main(String[] args) throws Exception {
+
+        ArrayList<Task> tasks = new ArrayList<>();
+        //scan for duke.txt file and add as input first
```

Comment:
Perhaps the placement of this comment is a bit off, it should be right before/ next to what it is trying to explaim. [See here](https://se-education.org/guides/conventions/java/basic.html#comments)